### PR TITLE
George

### PR DESCRIPTION
**Pull request by georgeb79**

### CODE REVIEW

**Review by Mariachaudhry — APPROVED** [review]
Approved

File: src/test/java/ServiceNow/NERD/Features/collaborationScenarios.feature [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
+  @SS-4265 @SS-4296 @juarezds @Regression
   Scenario: NIH strategic alignment (Secondary Optional) and HHS Strategic Plan Alignment (Secondary Optional) field should not be visible
     Given a Regular User has submitted a Collaboration
     When the DOC Planning Contact locates the record "Diego Test" in their Collaboration queue
     And clicks the Edit button for the record "Diego Test"
     And lands on the submission edit page for author "CBIIT Test Account"
     Then the Rank field is not visible and "Diego Test" collaboration is deleted for Automation Testing
 
-  @SS-4266 @juarezds @Regression
+  @SS-4264 @New @juarezds @Regression
```
Comment (zamant2) [inline src/test/java/ServiceNow/NERD/Features/collaborationScenarios.feature:11]: Any reason to use @new tag?

File: src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java [hunk 2]
```diff
@@ -42,8 +42,8 @@ public void data_submitted_for_scenario_one_is_verified_in_native_view_against_s
         rasScreenerStepsImpl.verifying_RAS_Screener_Scenario_1_Data();
 
         /******* VERIFY ENROLLMENT STATUS BELOW ********/
-        CommonUtils.clickOnElement(screenerRecordTablePage.dynamicLocatorForStudyButtons("Submit for Eligibility Review"));
-        CommonUtils.clickOnElement(screenerRecordTablePage.dynamicLocatorForStudyButtons("Mark Eligible"));
+//        CommonUtils.clickOnElement(screenerRecordTablePage.dynamicLocatorForStudyButtons("Submit for Eligibility Review"));
```
Comment (zamant2) [inline src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java:45]: We shouldn't merge commented lines.

File: src/test/java/ServiceNow/NERD/Features/collaborationScenarios.feature (hunk 1 above)
Comment (georgeb79) [reply src/test/java/ServiceNow/NERD/Features/collaborationScenarios.feature:11]: Removed

File: src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java (hunk 2 above)
Comment (georgeb79) [reply src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java:45]: Currently working on this one

File: src/test/java/ServiceNow/SEER/Pages/NativeViewEmailsPage.java [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 import com.nci.automation.web.CommonUtils;
 import com.nci.automation.web.WebDriverUtils;
+import org.openqa.selenium.By;
```
[the remark this answers is not in the document]
Comment (georgeb79) [reply src/test/java/ServiceNow/SEER/Pages/NativeViewEmailsPage.java]: Removed. Is not used.

**Review by zamant2 — APPROVED** [review]

**Review by sarwarahmed1 — APPROVED** [review]

File: src/test/java/ServiceNow/NERD/Features/ReturningSubmissions.feature [hunk 4]
```diff
@@ -29,7 +29,7 @@ Feature: Ruturning published Collaboration by the CRS Rreviewer/DOC Planning Con
     Given a Collaboration has been submitted to the DOC Planning Contact
     Then the submitted Collaboration shows as "Under Review" in the Submissions page
 
-  @SS-3895 @SS-3936 @bucurgb @Regression
+  @SS-3895 @SS-3936 @SS-3944 @SS-3982 @SS-3943 @SS-3986 @bucurgb @Regression @Progression
```
[the remark this answers is not in the document]
Comment (georgeb79) [reply src/test/java/ServiceNow/NERD/Features/ReturningSubmissions.feature]: Deleted

File: src/test/java/ServiceNow/NERD/Pages/NERDOGCRAddNewEntryPage.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
+package ServiceNow.NERD.Pages;
+
+import com.nci.automation.web.WebDriverUtils;
+import org.openqa.selenium.By;
```
[the remark this answers is not in the document]
Comment (georgeb79) [reply src/test/java/ServiceNow/NERD/Pages/NERDOGCRAddNewEntryPage.java:4]: Both are needed .By is needed for the dynamic XPath

**Review by hamidnesar — APPROVED** [review]

**Pull request merged**